doc/components/job-usage-calculation.rst

```diff
@@ -0,0 +1,85 @@
+.. _job-usage-calculation:
+
+#####################
+Job Usage Calculation
+#####################
+
+The raw job usage factor for an association is defined as the sum of products
+of number of nodes used (``nnodes``) and time elapsed (``t_elapsed``). To
+calculate the raw usage for a given association *U*:
+
+
+:math:`U = sum(nnodes \times t\_elapsed)`
+
+flux-accounting keeps track of job usage in a table according to two properties
+that are set when the database is first created: **PriorityDecayHalfLife** and
+**PriorityUsageResetPeriod**. Each of these parameters represent a number of
+weeks by which to hold usage factors up to the time period where jobs no longer
```

Comment:
Once I realized the decay was by period and not job this made way more sense. Nice work @cmoussa1!